src/mgxs_header.F90

```diff
@@ -496,6 +498,18 @@ subroutine mgxsiso_init_file(this, node_xsdata, groups, get_kfiss, get_fiss, &
                        (scatt_coeffs(imu - 1, gout, gin) + &
                         scatt_coeffs(imu, gout, gin))
                 end if
+                ! Now create CDF from fmu with the analytical integral of a
+                ! piecewise linear function
+                if (imu > 1) then
+                  p0 = scatt_coeffs(imu - 1, gout, gin)
+                  mu0 = mu - dmu
+                  m = (scatt_coeffs(imu, gout, gin) - &
+                       scatt_coeffs(imu - 1, gout, gin)) / dmu
+                  norm = norm + &
+                       HALF * m * mu * mu + &
+                       (p0 - m * mu0) * mu + &
+                       (HALF * m * mu0 * mu0 - p0 * mu0)
+                end if
```

Comment:
Doesn't this block replace the block immediately before? Looks like the previous one can be removed, no?